src/common/hooks/use-mount-effect.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useState } from 'react';
+
+export function useMountEffect() {
+  const [mounted, setMounted] = useState(false);
+  useEffect(() => {
+    setTimeout(() => {
```

Comment:
Should we consider using `requestAnimationFrame` instead of `setTimeout(() => {}, 0)`?

Reply:
done!